src/configuration/config.ts

```diff
@@ -19,14 +19,27 @@ const config: Config = {
   graphql: {
     debug: Env.isDev,
     introspection: Boolean(getEnv('INTROSPECTION_ENABLED')),
-    playgroundEnabled: false,
+    playground: {
+      settings: {
+        'request.credentials': 'include',
+      },
+    },
 
     cors: {
       credentials: true,
       originAsRegex: /https?:\/\/[a-z0-9\-_]+-fullstacks\.vercel\.app/,
       apolloSandboxOrigin: 'https://studio.apollographql.com',
     },
   },
+
+  auth: {
+    // https://github.com/expressjs/session
+    session: {
+      secret: getEnv('SESSION_SECRET', 'secret'),
```

Comment:
actually, it does not affect, anyway, Joi validation does validation before initializing this config object, so it's not affected anymore

Reply:
yes you are right. but why should we keep a code which has no affect?

Reply:
because `getEnv` return `string | undefined` as a value if we specify defaultValue it is just `string` returns, based on the session option type secret should be defined it's not optional so it is good practice to have this default value

Reply:
it is misleading. you can use `getEnv('SESSION_SECRET')!`

Reply:
I think it's clear to have a default value, let `ts` do it's job :))

Reply:
@MohammadForoutan @ASafaeirad 
what do you guys think?

Reply:
I don't agree with the default value. There are safer ways to bypass compile errors.

Reply:
done